x/consortium/client/cli/tx.go

```diff
@@ -151,18 +149,8 @@ $ %s tx gov submit-proposal update-validator-auths [flags]
 			}
 			deletingValidators := parseCommaSeparated(deletingValidatorsStr)
 
-			createAuths := func(addings, deletings []string) ([]*types.ValidatorAuth, error) {
+			createAuths := func(addings, deletings []string) []*types.ValidatorAuth {
 				var auths []*types.ValidatorAuth
-
-				// check duplications
-				usedAddrs := map[string]bool{}
-				for _, addr := range append(addings, deletings...) {
-					if usedAddrs[addr] {
-						return auths, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "multiple auths for same validator: %s", addr)
-					}
-					usedAddrs[addr] = true
-				}
-
 				for _, addr := range addings {
```

Comment:
remove the validation logic because we call `ValidateBasic()` which has the same logic